src/systemtraynotificationhandler.cpp

```diff
@@ -27,6 +28,9 @@ SystemTrayNotificationHandler::SystemTrayNotificationHandler(QObject* parent)
   MozillaVPN* vpn = MozillaVPN::instance();
   Q_ASSERT(vpn);
 
+  m_menu = new QMenu();
```

Comment:
`new QMenu(this)`

Reply:
This fails.

Reply:
ah right. QMenu wants QWidget as parent.
Let's use: `QScopedPointer<QMenu> m_menu`;